src/Arcus.Templates.Tests.Integration/Arcus - Backup.Templates.Tests.Integration.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Do we need this backup file ?

Reply:
Oh, now, must've slipped the git commit. I'll remove this. 👍